### ISSUE TITLE
/replaced: new command for answering AI tools taking jobs

### ISSUE THREAD

**Issue by damon314159** [TheOdinProject/odin-bot-v2#518]
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)

-   [x] The title of this issue follows the `command name: brief description of request` format, e.g. `/help: add optional @user parameter`

The following checkbox is OPTIONAL:
-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**
The same conversation happens day in day out, with youtubers and the like well understanding that spelling doom drives interaction, people are convinced that all software developers are to be replaced by AI in the near future, and that all their efforts to learn have been wasted. This conversation always yields the same, if not very similar, responses. A command like /replaced to summarise some of the key points could be really helpful to take the work off the helpers to answer these questions each time.

Happy to spend some time collating past responses to these questions and drafting up a message for the bot to deliver if staff think there's any merit to it.

**2. Acceptance Criteria:**
- [ ] New command added to give relevant comments

**Comment by damon314159** [comment]
Yeah, that makes sense. A proper explanation is probably a bit long for the body of a command, so a link to an existing message works. Like the time one

**Comment by bycdiaz** [comment]
I would love to see a command like this. I think it also merits a disclaimer. Something like: "These are our opinions and no one can tell the future, including the people on Reddit. Even us."

**Comment by JustWaveThings** [comment]
@damon314159 Thanks for raising this issue. Given discussions around slash command usage we've had recently, I am going to close this for now.
